libs/ui/src/config/features.ts

```diff
@@ -31,9 +34,7 @@ export function isCardReaderCheckDisabled(): boolean {
  */
 export function areVvsg2AuthFlowsEnabled(): boolean {
   return (
-    (process.env.NODE_ENV === 'development' ||
-      process.env.NODE_ENV === 'test' ||
-      asBoolean(process.env.REACT_APP_VX_DEV)) &&
+    (['development', 'test'].includes(process.env.NODE_ENV) || isVxDev()) &&
```

Comment:
should we remove is the dev/vxdev constraint here so we can actually build this into a production build?

Reply:
this is not required for this PR, to be clear.

Reply:
Yes good q! I'm going to enable VVSG2 auth flows by default in a separate PR, tracked by issue https://github.com/votingworks/vxsuite/issues/2171. I'm going to tackle that ASAP since this current PR is the last key functional PR for the project